tests/functional/clients/standalone/package_list/fixtures/alpine.py

```diff
@@ -0,0 +1,379 @@
+            "/var/local",
+            "/var/lock",
+            "/var/lock/subsys",
+            "/var/log",
+            "/var/mail",
+            "/var/opt",
+            "/var/spool/mail",
+            "/var/spool",
+            "/var/spool/cron/crontabs",
+            "/var/spool/cron",
+            "/var/tmp",
+        ],
+    },
+    "alpine-keys": {
+        "version": "2.2",
+        "sourcepkg": "alpine-keys",
+        "release": "r0",
+        "origin": "Natanael Copa <ncopa@alpinelinux.org>",
+        "arch": "x86_64",
+        "license": "MIT",
+        "size": "106496",
+        "type": "APKG",
+        "name": "alpine-keys",
+        "files": [
+            "/etc",
+            "/etc/apk",
+            "/etc/apk/keys/alpine-devel@lists.alpinelinux.org-4a6a0840.rsa.pub",
+            "/etc/apk/keys/alpine-devel@lists.alpinelinux.org-5243ef4b.rsa.pub",
+            "/etc/apk/keys/alpine-devel@lists.alpinelinux.org-5261cecb.rsa.pub",
+            "/etc/apk/keys",
+            "/usr",
+            "/usr/share",
+            "/usr/share/apk",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-4a6a0840.rsa.pub",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-5243ef4b.rsa.pub",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-524d27bb.rsa.pub",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-5261cecb.rsa.pub",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-58199dcc.rsa.pub",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-58cbb476.rsa.pub",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-58e4f17d.rsa.pub",
+            "/usr/share/apk/keys/alpine-devel@lists.alpinelinux.org-5e69ca50.rsa.pub",
+            "/usr/share/apk/keys",
+            "/usr/share/apk/keys/aarch64/alpine-devel@lists.alpinelinux.org-58199dcc.rsa.pub",
+            "/usr/share/apk/keys/aarch64",
+            "/usr/share/apk/keys/armhf/alpine-devel@lists.alpinelinux.org-524d27bb.rsa.pub",
+            "/usr/share/apk/keys/armhf",
+            "/usr/share/apk/keys/mips64/alpine-devel@lists.alpinelinux.org-5e69ca50.rsa.pub",
+            "/usr/share/apk/keys/mips64",
+            "/usr/share/apk/keys/ppc64le/alpine-devel@lists.alpinelinux.org-58cbb476.rsa.pub",
+            "/usr/share/apk/keys/ppc64le",
+            "/usr/share/apk/keys/s390x/alpine-devel@lists.alpinelinux.org-58e4f17d.rsa.pub",
+            "/usr/share/apk/keys/s390x",
+            "/usr/share/apk/keys/x86/alpine-devel@lists.alpinelinux.org-4a6a0840.rsa.pub",
+            "/usr/share/apk/keys/x86/alpine-devel@lists.alpinelinux.org-5243ef4b.rsa.pub",
+            "/usr/share/apk/keys/x86",
+            "/usr/share/apk/keys/x86_64/alpine-devel@lists.alpinelinux.org-4a6a0840.rsa.pub",
+            "/usr/share/apk/keys/x86_64/alpine-devel@lists.alpinelinux.org-5261cecb.rsa.pub",
+            "/usr/share/apk/keys/x86_64",
+        ],
+    },
+    "busybox": {
```

Comment:
busybox? That is surprising... I guess this is an actual package via `apkg`?

Reply:
indeed it is